### DISCUSSION TITLE
Dongle Remote SONOFF ZBDongle-E via ser2net

### DISCUSSION THREAD

**Discussion by matzeeg3** [Koenkk/zigbee2mqtt#16552] [General]
Hi everbody,
i have an SONOFF ZBDongle-E and i know its experimental right now.
I use Docker for Zigbee2MQTT and if i plug the Stick directly to the Host and map the device it worked but i need the Stick on a different place in my house so i have a host with ser2net running and use the configuration provided by the docs.
https://www.zigbee2mqtt.io/advanced/remote-adapter/connect_to_a_remote_adapter.html
i checked the connection via ncat
`nc -z -v -w 1 ip 20108` 
and i get an "ok" but if i change the configuration.yml and start zigbee2mqtt it says timeout for the stick.
is there a special config for ZBDongle-E in ser2net or what am i doing wrong?
here is the log from the docker continer:
`zigbee2mqtt  | Zigbee2MQTT:error 2023-02-06 08:29:47: Error while starting zigbee-herdsman
zigbee2mqtt  | Zigbee2MQTT:error 2023-02-06 08:29:47: Failed to start zigbee  
zigbee2mqtt  | Zigbee2MQTT:error 2023-02-06 08:29:47: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
zigbee2mqtt  | Zigbee2MQTT:error 2023-02-06 08:29:47: Exiting...
zigbee2mqtt  | Zigbee2MQTT:error 2023-02-06 08:29:47: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
zigbee2mqtt  |     at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:103:27)
zigbee2mqtt  |     at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
zigbee2mqtt  |     at Zigbee.start (/app/lib/zigbee.ts:58:27)
zigbee2mqtt  |     at Controller.start (/app/lib/controller.ts:101:27)
zigbee2mqtt  |     at start (/app/index.js:107:5)`

**Comment by matzeeg3** [comment]
hey i fixed it by my own, for everbody how has the same problem just add:
adapter: ezsp
to the serial part of the configuration.yml

**Comment by grid8** [comment]
Thanks, that saved me some time. My LQI on identical devices in the same locations doubled at minimum and often tripled going from the P to E series coordinator.
